tripal_bulk_loader/includes/tripal_bulk_loader.loader.inc

```diff
@@ -164,10 +164,14 @@ function tripal_bulk_loader_load_data($nid, $job_id) {
     $total_lines++;
   }
   fclose($handle);
+  // do not count header line in the non-blank line count
+  if (($total_lines) and (preg_match('/(t|true|1)/', $node->file_has_header))) {
```

Comment:
Could you do this instead:
```
if (($total_lines) and ($node->file_has_header === TRUE)) {
```
The `preg_match` is meant for strings but I would think that `$node->file_has_header` is a boolean?

Reply:
No, for some reason it's not. var_dump says for $node->file_has_header
```string(1) "1"```
I just copied that exact logic from pre-pull line # 335

Reply:
It is super-ultra-not-boolean! If I set File has header to "No" it is now
```string(1) "2"```

Reply:
I checked and the "No" value is never tested for.
Just because this bothers me a tiny bit I am going to change No from a 2 to a zero.